pact/Pact/Core/Compile.hs

```diff
@@ -229,11 +228,13 @@ evalTopLevel interpreter (RawCode code) tlFinal deps = do
       mSize <- sizeOf (_mInfo m) SizeOfV0 mdata
       chargeGasArgs (_mInfo m) (GWrite mSize)
       evalWrite (_mInfo m) pdb Write DModules (view mName m) mdata
+
+      evalWrite (_mInfo m) pdb Write DModuleSource (getHashedModuleName m) (ModuleCode code)
+
       -- Get all
       let newTopLevel = M.fromList $ (\(fqn, d) -> (_fqName fqn, (fqn, defKind (_mName m) d))) <$> fqDeps
           loadNewModule =
             over loModules (M.insert (_mName m) mdata) .
-            over loAllLoaded (M.union newLoaded) .
```

Comment:
This line was removed by the way since on line 222, the following happens:
```
-- working set uses `lookupFqNameOrFail` and it _will_ lookup these deps
      esLoaded . loAllLoaded %= M.union newLoaded
```

So it is redundant.